data/records.json

```
@@ -40,5 +40,26 @@
         "company_site": "",
         "license": "freeware",
         "upload_date": "08/11/2021"
+    },
+    {
+        "name": "Bot - Google Maps Routes",
+        "description_short": "Robot to search for routes on google maps and collect information.",
+        "description": "Robot to search for routes on google maps and collect information. ",
+        "tags": ["web"],

```

This bot is Desktop not Web despite using Chrome.
```suggestion
        "tags": ["desktop"],
```